File: djCSharp/HelloCSharp009/HelloCSharp009/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HelloCSharp009
{
    public class Student<T>
    {
        public T hakbeon { get; set; }
        public string name { get; set; }
        public int age { get; set; }
    }
}

File: Python/djPy_djChat/mydjpy/pythonProject4/Student.py
class Student:
    def __init__(self,name,age):
        self.name = name
        self.age = age
    def __del__(self):
        print('소멸됩니다')
    def introduce(self):
        print('이름 {}, 나이 {}'.format(self.name,self.age))